bundle/manifests/mtk-operator.clusterserviceversion.yaml

```diff
@@ -36,6 +36,18 @@ spec:
     spec:
       clusterPermissions:
       - rules:
+        - apiGroups:
```

Comment:
I wonder if we can scope this to a well-known name for our specific plugin?

Reply:
+1, thanks for pointing this out.

How about `crane-ui-plugin` for a name?

Reply:
Sounds good to me